src/github-issues-prs.ts

```diff
@@ -288,3 +305,10 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 		return remotes;
 	}
 }
+
+/**
+ * Helper function which removes the ending `.git`
+ */
+function getGitHubUrl(url: string): string {
+	return url.substr(0, url.lastIndexOf('.git'));
```

Comment:
The URL might be in SSH format (e.g., git@github.com:octocat/Hello-World.git). You can instead use `github.repos.get(...)` (see https://developer.github.com/v3/repos/#get) to get the repository's data and use `html_url` of that.